charts/gha-runner-scale-set-controller/values.yaml

```diff
@@ -3,6 +3,8 @@
 # Declare variables to be passed into your templates.
 labels: {}
 
+podTemplateLabels: {}
```

Comment:
Could you move this below the podAnnotations and also name it `podLabels`?

Reply:
Done. 
Thanks a lot @nikola-jokic !

Reply:
I'll run some tests just to make sure nothing breaks, but looks good to me! Thank you for your contribution!